helphours/stats.py

```diff
@@ -123,9 +123,8 @@ def generate_graphs(range):
     filtered.insert(4, "min_in_line", min_in_line)
 
     # obtain relevant data for only last week, last month
-    today = datetime.today().date()
-    week_prior = today - timedelta(days=7)
-    month_prior = today - timedelta(days=30)
+    week_prior = pd.Timestamp('now').floor('D') + pd.offsets.Day(-7)
+    month_prior = pd.Timestamp('now').floor('D') + pd.offsets.Day(-30)
```

Comment:
Hmmmm forgot to consider leap years...

Just kidding looks good!

Reply:
F